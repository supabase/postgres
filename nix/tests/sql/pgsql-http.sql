select
  urlencode('my special string''s & things?');

select
  content
from
  http_get (
    'https://postman-echo.com/get?foo1=bar1&foo2=bar2'
  );
