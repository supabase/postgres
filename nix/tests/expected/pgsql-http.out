select
  urlencode('my special string''s & things?');
              urlencode              
-------------------------------------
 my+special+string%27s+%26+things%3F
(1 row)

select
  content
from
  http_get (
    'https://postman-echo.com/get?foo1=bar1&foo2=bar2'
  );
ERROR:  OpenSSL/3.0.13: error:16000069:STORE routines::unregistered scheme
